pkg/controller/master/backup/backup.go

```diff
@@ -366,7 +368,7 @@ func (h *Handler) reconcileVolumeSnapshots(vmBackup *harvesterv1.VirtualMachineB
 		if volumeSnapshot == nil {
 			volumeSnapshot, err = h.createVolumeSnapshot(vmBackupCpy, volumeBackup)
 			if err != nil {
-				logrus.Debugf("create volumeSnapshot %s/%s error: %v", volumeSnapshot.Namespace, volumeSnapshot.Name, err)
```

Comment:
when err, volumeSnapshot is nil, access volumeSnapshot.Namespace caused panic @FrankYang0529

Reply:
Yes, you're right. Thank you!